Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/RelatedFigure/GetRelatedFiguresByStreetcodeIdTests.cs

```diff
@@ -0,0 +1,207 @@
+        {
+            var repository = new Mock<IRepositoryWrapper>();
+            var mockMapper = new Mock<IMapper>();
+
+            var testRelatedList = new List<Entities.RelatedFigure>()
+            {
+                    new Entities.RelatedFigure()
+                    {
+                        ObserverId = id,
+                        TargetId = id + 1,
+                        Observer = new StreetcodeContent(){},
+                        Target = new StreetcodeContent(){}
+                    }
+            };
+
+            var testPersonStreetcodeList = new List<PersonStreetcode>()
+            {
+                new PersonStreetcode()
+                {
+                    FirstName= "Test",
+                    LastName = "TestSurname",
+                    Id = id,
+                    Index = id,
+                    Teaser = "test",
+                    EventStartOrPersonBirthDate = DateTime.UtcNow,
+                    EventEndOrPersonDeathDate = DateTime.UtcNow.AddDays(1),
+                }   
+            };
+
+            var testRelatedDTOList = new List<RelatedFigureDTO>()
+            {
+                new RelatedFigureDTO() { Id = id }
+            };
+
+
+
+            repository.Setup(x => x.RelatedFigureRepository
+                .FindAll(It.IsAny<Expression<Func<Entities.RelatedFigure, bool>>?>()))
+                .Returns(testRelatedList.AsQueryable());
+
+            repository.Setup(x => x.StreetcodeRepository.GetAllAsync(
+                It.IsAny<Expression<Func<StreetcodeContent, bool>>?>(),
+                It.IsAny<Func<IQueryable<StreetcodeContent>, IIncludableQueryable<StreetcodeContent, object>>?>()))
+                .ReturnsAsync(testPersonStreetcodeList);
+
+            mockMapper.Setup(x => x.Map<IEnumerable<RelatedFigureDTO>>(It.IsAny<IEnumerable<object>>()))
+            .Returns(testRelatedDTOList);
+
+            var handler = new GetRelatedFiguresByStreetcodeIdHandler(mockMapper.Object, repository.Object);
+
+            var result = await handler.Handle(new GetRelatedFigureByStreetcodeIdQuery(id), CancellationToken.None);
+
+            Assert.NotNull(result.Value);
+            Assert.NotEmpty(result.Value);
+        }
+
+        [Theory]
+        [InlineData(1)]
+        public async Task GetRelatedFigureByStreetcodeId_Type_Check(int id)
+        {
+            var repository = new Mock<IRepositoryWrapper>();
+            var mockMapper = new Mock<IMapper>();
+
+            var testStreetcodeContentList = new List<StreetcodeContent>()
+            {
+                new StreetcodeContent()
+                {
+                    Id = 1,
+                    Index = 1,
+                    Teaser = "test",
+                    EventStartOrPersonBirthDate = DateTime.UtcNow,
+                    EventEndOrPersonDeathDate = DateTime.UtcNow.AddDays(1),
+                    Images = new List<Image>{ new Image() { Id = 1} },
+                    Tags = new List<Tag>{new Tag() { Id = 1} }
```

Comment:
If you use inline data let's pass all parameters

Reply:
same here, removed most of it